defaults/severities.yaml

```diff
@@ -18,6 +18,7 @@ FullPartitionScanReversedOrderEvent: ERROR
 FullPartitionScanEvent: ERROR
 TombstoneGcVerificationEvent: ERROR
 FullScanAggregateEvent: ERROR
+DatabaseLogEvent.OVERSIZE_ALLOCATION: ERROR
```

Comment:
Please, add letter `D` to the `OVERSIZE` word.
Without it it looks like a command for action and not happened fact.